.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release Chart
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
    branches:
      - main
    paths:
      - 'charts/**'
```

Reply:
Does it make sense to run only when chart files change?

Reply:
Based on our configuration, the charts files change once this action has run, since users shouldn't push to charts directly

Reply:
doh, yeah of course. nevermind this suggestion then.